### PR TITLE
[FEAT] Rubies

### PR DESCRIPTION
**Pull request by Rofrtd**
# Description

This PR implements the new mineral Ruby, Ruby Stone and Gold Pickaxe.

Obs.: Assets are placeholders while Design Team works on getting the final assets ready.

Fixes #issue

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [ ] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [ ] Screenshot if it includes any UI changes
- [ ] I have read the contributing guidelines and agree to the T&Cs
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: metadata/metadata.ts [hunk 1]
```diff
@@ -462,6 +462,18 @@ export const OPEN_SEA_ITEMS: Record<InventoryItemName, Metadata> = {
       { trait_type: "Tradable", value: "No" },
     ],
   },
+  "Gold Pickaxe": {
+    name: "Gold Pickaxe",
+    description:
+      "A tool used to mine rubies. It is burnt after use.\n\nYou can craft an gold pickaxe at the Blacksmith in the game.",
```
Comment (brian-man) [inline metadata/metadata.ts:468]: Minor: "an gold" -> "a gold"

File: src/features/game/expansion/lib/expansionNodes.ts [hunk 2]
```diff
@@ -10,6 +10,7 @@ export interface Nodes {
   "Stone Rock": number;
   "Iron Rock": number;
   "Gold Rock": number;
+  "Ruby Rock": number;
```
Comment (brian-man) [inline src/features/game/expansion/lib/expansionNodes.ts:13]: These data structures are used for lands that haven't prestiged yet so they shouldn't have Ruby Rock, right?

File: src/features/game/types/tools.ts [hunk 3]
```diff
@@ -57,6 +57,16 @@ export const WORKBENCH_TOOLS: (
       Iron: new Decimal(5),
     },
   },
+  "Gold Pickaxe": {
+    name: "Iron Axe",
+    description: "Used to collect gold",
```
Comment (brian-man) [inline src/features/game/types/tools.ts:62]: gold -> rubies

File: src/features/game/types/tools.ts [hunk 4]
```diff
@@ -57,6 +57,16 @@ export const WORKBENCH_TOOLS: (
       Iron: new Decimal(5),
     },
   },
+  "Gold Pickaxe": {
+    name: "Iron Axe",
```
Comment (brian-man) [inline src/features/game/types/tools.ts:61]: Iron Axe -> Gold Pickaxe

File: src/features/game/types/tools.ts [hunk 5]
```diff
@@ -57,6 +57,16 @@ export const WORKBENCH_TOOLS: (
...
+    description: "Used to collect gold",
+    sfl: new Decimal(0.5),
+    tokenAmount: new Decimal(10),
```
Comment (brian-man) [inline src/features/game/types/tools.ts:64]: Should `tokenAmount` be here? It's not in the other tools.

File: src/features/game/expansion/lib/expansionNodes.ts (hunk 2 above)
Comment (Rofrtd) [reply src/features/game/expansion/lib/expansionNodes.ts:13]: Yep, these are for testing.. 
Draft PR.

**Review by adamhannigan — DISMISSED** [review]
LGTM ✅ Just a few backend considerations

**Pull request merged**